pkg/apis/objectbucket.io/v1alpha1/objectbucket_types.go

```diff
@@ -37,7 +39,9 @@ const (
 
 // ObjectBucketStatus defines the observed state of ObjectBucket
 type ObjectBucketStatus struct {
-	Controller *v1.ObjectReference
+	Controller *v1.LocalObjectReference
```

Comment:
is including the controller in status fairly typical?

Reply:
The sig-storage pattern is to add an annotation to the object that the controller "claims" responsibility for.  This seems a little cleaner than mutating / analyzing the annotations map[string]string.